File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_6.py
sobrenome = input('Digite sobrenome do apresentador:').upper()

if sobrenome == 'PINHEIRO' or sobrenome == 'ARAUJO' or sobrenome == 'CARLA':
    print('Jornal: Bom Dia Nação.')
elif sobrenome == 'BONNER' or sobrenome == 'VASCONCELOS':
    print('Jornal: Jornal Brasileiro.')
else:
    print('Apresenta-dor(a) desconhecido(a).')

File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-6.js
prompt = require('prompt-sync')();

sobrenome = prompt('Digite o sobrenome do apresentador: ').toUpperCase()

if(sobrenome == 'PINHEIRO' || sobrenome == 'ARAUJO'){
    console.log('Jornal: Bom Dia Nação')
}else if(sobrenome == 'BONNER' || sobrenome == 'VASCONCELOS'){
    console.log('Jornal: Jornal Brasileiro')
}else{
    console.log('apresenta-dor(a) desconhecido(a).')
}